content/en/Features/db_migration.md

```diff
@@ -0,0 +1,257 @@
+```
+
+As long as the migrations are short, this option should work fine, keep your database and server changes in lock step 
+and keep your ci/cd unchanged.
+
+### In a Running Server's Container
+An alternative route is having a user enter one of the running containers via the `opta shell` command and execute the 
+migration manually. This option offers more developer control, but now requires a human in the loop, does not enforce
+synchronicity between the database and server versions, and users will need to be careful not to trigger a new deployment
+while the operation is taking place.
+
+### In a One-Time Job
+For a cleaner separation of migration and server deployment, you can also run a migration as a separate 
+[kubernetes job](https://kubernetes.io/docs/concepts/workloads/controllers/job/) by using a helm chart module
+to create the job with the required inputs. This can be done in either the existing service's manifest, or a new 
+manifest which refers to the current manifest for data such as the current container image deploy (we recommend the latter
+approach). Take for example the following manifests which starts with a k8s service with a database and then uses a known 
+public helm chart which creates a one-time job for the migration in either the same manifest or a new one:
+
+{{< tabs tabTotal="2" >}}
+{{< tab tabName="New Manifest" >}}
+
+```yaml
+environments:
+  - name: staging
+    path: "./staging.yaml"
+name: my-service-migration
+modules:
+  - type: external-state
+    name: external
+    backend_type: s3
+    config:
+      bucket: opta-tf-state-my-org-staging
+      key: my-service
+      region: us-east-1
+  - type: helm-chart
+    name: dbmigration1
+    namespace: my-service
+    create_namespace: false
+    repository: https://ameijer.github.io/k8s-as-helm/
+    chart: job
+    chart_version: 1.0.0
+    values:
+      nameOverride: "databaseMigration"
+      restartPolicy: Never
+      containers:
+        hello:
+          image: "${{module.external.outputs.current_image}}"
+          extraSettings:
+            env:
+              - name: DBHOST
+                valueFrom:
+                  secretKeyRef:
+                    name: secret
+                    key: DBHOST
+                    optional: true
+              - name: DBNAME
+                valueFrom:
+                  secretKeyRef:
+                    name: secret
+                    key: DBNAME
+                    optional: true
+              - name: DBPASS
+                valueFrom:
+                  secretKeyRef:
+                    name: secret
+                    key: DBPASS
+                    optional: true
+              - name: DBUSER
+                valueFrom:
+                  secretKeyRef:
+                    name: secret
+                    key: DBUSER
+                    optional: true
+            command:
+              - ./home/app/do_migration.sh
+```
+{{< /tab >}}
+{{< tab tabName="Same Manifest" >}}
```

Comment:
this solution seems simpler, having everything define in the same place makes it easy to understand.
why not only keep this one? or at least place it first.
the other looks more advanced

Reply:
The job needs to be renamed on every run, which can cause unpleasant complications/errors on regular deploys, which is why I think it ought to be a separate service

Reply:
I placed it first now